src/main/anovos/data_ingest/ts_auto_detection.py

```diff
@@ -37,6 +37,27 @@
 import pandas as pd
 import numpy as np
 
+
+def output_to_local(output_path):
+    """
+
+    Parameters
+    ----------
+    output_path :
+
+
+    Returns
+    -------
+
+    """
+    punctuations = ":"
+    for x in output_path:
+        if x in punctuations:
+            local_path = output_path.replace(x, "")
+            local_path = "/" + local_path
+    return local_path
+
+
```

Comment:
rather than repeating the same function def in 2 places, can you move it to utils and define it once and import it from utils and use ?

Reply:
Also rather than leaving the docstrings empty 
can we add meaningful examples in this case 
Like if dbfs://sample_path
is passed
then output returns 
/dbfs//sample_path ?